requirements.txt

```diff
@@ -10,7 +10,3 @@ requests==2.31.0
 tqdm==4.66.1
 sacrebleu==2.4.0
 datasets==2.17.1
-langchain==0.1.6
-langchain-community==0.0.20
-langchain-core==0.1.45
```

Comment:
langchain is still used here: clarifai_model_utils/llm_eval/evaluator/harness_eval/python_templates/ragas_eval.py

Reply:
Langchain is installed along with Ragas. I want to synchronize them to avoid dependency conflicts.